src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskInitiateByIdControllerTest.java

```diff
@@ -377,7 +382,10 @@ public void should_return_a_201_when_initiating_a_specific_access_task_by_id() {
                     "roleAssignmentId", "roleAssignmentId")))
                 .body("task.minor_priority", equalTo(500))
                 .body("task.major_priority", equalTo(1000))
-                .body("task.priority_date", equalTo("2022-12-07T13:00:00+0000"));
+                .body("task.priority_date", equalTo("2022-12-07T13:00:00+0000"))
+                .body("task.last_updated_timestamp", notNullValue())
+                .body("task.last_updated_user", equalTo(idamSystemUser))
+                .body("task.last_updated_action", equalTo(CONFIGURE.getValue()));
         };
 
         initiateTask(taskVariables, caseworkerCredentials.getHeaders(), assertConsumer);
```

Comment:
what about AC1? I.e., configure task and autoassigned and 
last_updated_timestamp: Timestamp of the update to the task
last_updated_user: IDAM ID of authenticated user (system user)
last_update_action: 'AutoAssign'